packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -263,6 +274,9 @@ export const coinjoinReducer = (
             case COINJOIN.ACCOUNT_CREATE:
                 createAccount(draft, action.payload);
                 break;
+            case COINJOIN.ACCOUNT_SET_LIQUIDITY_CLUE:
```

Comment:
How often is coinjoin account stored into DB? Shouldn't it be stored after this action as well?

Reply:
good catch, it should

Reply:
https://github.com/trezor/trezor-suite/pull/7231/commits/ef5072db0064f8adfef030c46f53fb9992efc5fb